golem-api-grpc/proto/golem/template/template_service.proto

```diff
@@ -15,6 +15,7 @@ service TemplateService {
   rpc GetTemplate (GetTemplateRequest) returns (GetTemplateResponse);
   rpc GetLatestTemplateVersion (GetLatestTemplateVersionRequest) returns (GetLatestTemplateVersionResponse);
   rpc UpdateTemplate (stream UpdateTemplateRequest) returns (UpdateTemplateResponse);
+  rpc GetVersionedTemplate(GetVersionedTemplateRequest) returns (GetVersionedTemplateResponse);
```

Comment:
I think it would be better with the following name changes:
- `GetVersionedTemplate` => `GetTemplateMetadata`
- `GetTemplate` => `GetTemplateMetadataAllVersions`

Reason: these are only returning template metadata, not the template itself. Also all templates are "versioned" so it's not a good way to distinguish between the two

Reply:
Another thing: until now the gRPC API and the REST API were the same - just two versions of the same API. I would like to keep that, so let's add this new endpoint to the REST API too.

Reply:
Sure